spec/latest/core/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
 <title>Hydra Core Vocabulary</title>
 <meta charset="utf-8">
+<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch|gated"></script>
```

Comment:
Re: the dependency - you could use pure XHR instead of fetch. Just requires more code but is doable. The polyfill would be required anyway, because ReSpec internally requires Promises. On the other hand ReSpec doesn't support IE anyways ATM so we could just drop that anyway actually

Re: more requests (speaking about imports not poylfill) - not sure if it's a big deal. They will all be small and nicely cacheable. 

And doesn't ReSpec have a `build` facitily to combine all those files together in a ready to publish form?